### ISSUE TITLE
[feature request] midi export

### ISSUE THREAD

**Issue by teetteet** [ja-k-e/random-commander#1]
can random_commander export midi files or can you implement it?

**Comment by ja-k-e** [comment]
this has been one of the main goals for random_commander. https://github.com/sergi/jsmidi seems like it may be outdated, but I'm going to look into https://github.com/nfroidure/MIDIFile. thanks for the request, now i know someone else really wants that functionality too.

**Comment by teetteet** [comment]
jsmidi is 4 years old, but reading the descriptions about writing to Base64 string sounds really good.
thumbs up for reaching that goal.

**Comment by ja-k-e** [comment]
yeah i saw that, i'm debating whether or not i should just concoct my own barebones conversion based on that. thanks!

**Comment by OliverBalfour** [comment]
I really think that you should add some way to export the audio. It's really cool, and for the retro game I'm making at the moment it generates music absolutely perfect, and I don't really want to have to mess around with the source code for this as I'm clueless as to how a fair chunk of it works. Being able to export it to any kind of file would make it an extremely useful tool.
